### PR TITLE
bump terpnetworktestnet 

### PR DESCRIPTION
**Pull request by discoverdefiteam**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Please archive (make new directory with chain.json and assetlist.json) the old network (90u-2) as either terpnetworktestnet1 (because first increment) or terpnetworktestnet2 (2 because the chain id was 90u-2). This new terpnetworktestnet (90u-4) should have a fresh version history starting at v4.2.0). I don't have permissions to edit your branch to help out.

**Comment by discoverdefiteam** [comment]
should the status of the archive be changed to killed?

File: testnets/terpnettestnet2/chain.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "terpnettestnet",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet2/chain.json:3]: "chain_name": "terpnettestnet2"

File: testnets/terpnettestnet2/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "terpnettestnet",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet2/assetlist.json:3]: "chain_name": "terpnettestnet2"

**Pull request merged**